openedx_events/event_bus/avro/tests/test_avro.py

```diff
@@ -48,6 +48,7 @@ def generate_test_event_data_for_data_type(data_type):
         int: 1,
         bool: True,
         str: "default",
+        float: 1.0,
```

Comment:
Mmm, can you explain a bit more why did you have to change this?

Reply:
Great! Thanks for the explanation. Hello, @whuang1202! I noticed you implemented these tests. Does this change make sense to you?